src/overlays/actors/ovl_En_Dg/z_en_dg.h

```diff
@@ -5,12 +5,32 @@
 
 struct EnDg;
 
-typedef void (*EnDgActionFunc)(struct EnDg* this, GlobalContext* globalCtx);
+typedef void (*EnDgActionFunc)(struct EnDg*, GlobalContext*);
+
+#define ENDG_GET_FC00(thisx) (((thisx)->params & 0xFC00) >> 0xA)
+#define ENDG_GET_3E0(thisx) (((thisx)->params & 0x3E0) >> 5)
```

Comment:
```suggestion
#define ENDG_GET_PARAMS_FC00(thisx) (((thisx)->params & 0xFC00) >> 0xA)
#define ENDG_GET_PARAMS_3E0(thisx) (((thisx)->params & 0x3E0) >> 5)
```

Reply:
Kenix said the prototype is just ovl_get_bits.